### ISSUE TITLE
minimize,maximize,close icons are now black? losing focus and going back to the app returns the white icons

### ISSUE THREAD

**Issue by Void48** [minusium/MicaForEveryone#27]
![image](https://user-images.githubusercontent.com/85196642/152468935-7288d360-d6ab-4576-8f3e-6d7ee69a8530.png)
![image](https://user-images.githubusercontent.com/85196642/152469103-a62f6ab5-47e9-45c6-9f05-8481e9c46998.png)
v0.0.4.1 doesn't have this issue

**Comment by minusium** [comment]
I know, I had same issue on 0.0.4.1 too and I thought it's a dwm bug. Maybe it's because I migrated to UI Automation so now it applies rule after window is shown, while before that it used to apply after window is created (and maybe it's before window being shown)
For workaround I tried to send SWP_FRAMECHANGED message but didn't work.
I can hide window and show it again after rule applied but i think it's not ideal. Idk.
